docs/src/pages/customization/components/components.md

```diff
@@ -99,35 +52,35 @@ In order to override the components special states, **you need to increase speci
 Here is an example with the _disable_ state and the button component using a **pseudo-class** (`:disabled`):
 
 ```css
-.Button {
+.Slider {
   color: black;
 }
 /* Increase the specificity */
-.Button:disabled {
+.Slider:disabled {
   color: white;
 }
 ```
 
 ```jsx
-<Button disabled className="Button">
+<Slider disabled className="Slider">
 ```
 
 Sometimes, you can't use a **pseudo-class** as the state doesn't exist in the platform.
```

Comment:
Any idea what "platform" refers to here?

Reply:
The user-agent